appcontainer/ViewportSizeModel.js

```diff
@@ -18,10 +19,15 @@ export class ViewportSizeModel extends HoistModel {
     /** @member {Object} - contains `width` and `height` in pixels */
     @observable.ref size;
 
+    /** @member {int} - orientation in degrees [0, 90,180, 270] */
+    @observable.ref orientation;
+
     /** @returns {boolean} */
     @computed
     get isPortrait() {
-        return this.size.width < this.size.height;
+        const {orientation, size} = this;
```

Comment:
this is a bit of a fragile `computed`- - it is relying on the fact that we dereference the size on line 28.  If we moved that down past the next line, our observability would break.  Probably need a comment if we want to let size drive the observability.

In general, am wondering two things: 

1) Should we skip the public observable 'orientation'? Seems like it kind of conflicts with isPortrait, and its a deprecated API.

2) Perhaps instead within isPortrait, we can just check the new orientation, and then the deprecated orientation, and then the width < height

Reply:
Great suggestions - will make these change 👍